### ISSUE TITLE
Model torso slooooooooooooowly rotates

### ISSUE THREAD

**Issue by ExpiredPopsicle** [ExpiredPopsicle/SnekStudio#74]
If you leave the software on for an entire day, the model can end up slightly rotated.

**Comment by copygirl** [comment]
Unsure if related, but if you manually move the tracker objects quite a bit out of their usual range, the IK code (?) causes the body to end up rotated, and it does not return to normal.

**Comment by ExpiredPopsicle** [comment]
Yeah I guessed it was that plus possibly accumulating floating point error.
